### PR TITLE
Added Lore Mastery to 20170213

### PR DESCRIPTION
**Pull request by NickVendel**
#44

### CODE REVIEW

File: unearthed-arcana/2017/20170213.xml [hunk 1]
```diff
@@ -163,5 +163,90 @@
...
+            <grant type="Archetype Feature" name="ID_WOTC_UA_ARCHETYPE_LORE_MASTERY_LORE_MASTER" level="2"/>
+            <grant type="Archetype Feature" name="ID_WOTC_UA_ARCHETYPE_LORE_MASTERY_SPELL_SECRETS" level="2"/>
+            <grant type="Archetype Feature" name="ID_WOTC_UA_ARCHETYPE_LORE_MASTERY_ALCHEMICAL_CASTING" level="6"/>
+            <grant type="Archetype Feature" name="ID_WOTC_UA_ARCHETYPE_LORE_MASTERY_PRODIGIOUS_MEMORY" level="10"/>
+            <grant type="Archetype Feature" name="ID_WOTC_UA_ARCHETYPE_LORE_MASTERY_MASTER_OF_MAGIC" level="14"/>
+        </rules>
+    </element>
+    <element name="Lore Master" type="Archetype Feature" source="Unearthed Arcana" id="ID_WOTC_UA_ARCHETYPE_LORE_MASTERY_LORE_MASTER">
+        <description>
+            <p>Starting at 2nd level, you become a compendium of knowledge on a vast array of topics. Your proficiency bonus is doubled for any ability check you make that uses the Arcana, History, Nature, or Religion skill if you are proficient in that skill.</p>
```
[the remark this answers is not in the document]
Comment (NickVendel) [reply unearthed-arcana/2017/20170213.xml]: You're right. I will edit it.

[the remark this answers is not in the document]
Comment (NickVendel) [reply unearthed-arcana/2017/20170213.xml]: Fixed it. Waiting for the review.

[the remark this answers is not in the document]
Comment (swdriessen) [reply unearthed-arcana/2017/20170213.xml]: just add them like so:

`<stat name="Arcana Proficiency" value="Proficiency" bonus="double" requirements="ID_PROFICIENCY_SKILL_ARCANA" />`

best not to use these workaround element when they are not needed

[the remark this answers is not in the document]
Comment (NickVendel) [reply unearthed-arcana/2017/20170213.xml]: Done.

**Pull request merged**